cli/cmd/root.go

```diff
@@ -265,6 +268,18 @@ func (options *proxyConfigOptions) validate() error {
 			options.proxyLogLevel)
 	}
 
+	if options.rawIgnoreInboundPorts != nil {
```

Comment:
I think (correct me if wrong) this function only validates things, but now it not only validates it but it mutates the options as well. So we can either move that logic to another function or simply rename it to `parseAndValidate` ?

Reply:
Not a problem to change the name, I hadn't really thought of it that way but it does make sense.